hash-parser/src/parse.rs

```diff
@@ -13,10 +13,37 @@ pub fn module(source: &str) -> Result<Module, ParseError> {
 
     match result {
         Ok(_pairs) => {
-            // here is where we do all the ast-ification
+            // @@Incomplete: here is where we do all the ast-ification
             let vec = Vec::new();
             Ok(Module { contents: vec })
         }
         Err(err) => Err(ParseError::from(err)),
     }
 }
+
+/// Function to parse an individual statement. This function is primarily used for the interactive
+/// mode where only statements are accepted.
+pub fn statement(source: &str) -> Result<AstNode<Statement>, ParseError> {
+    let result = HashParser::parse(Rule::statement, source);
```

Comment:
You can use the question mark operator `let result = HashParser::parse(Rule::statement, source)?;` to prevent having to match.